gadk/elements.py

```diff
@@ -162,11 +202,20 @@ def __init__(
         if default_checkout:
             self._steps.insert(0, UsesStep(action=ACTION_CHECKOUT))
 
+    def __repr__(self):
+        repr_ = f"<{type(self).__name__}"
+        if self._name is not None:
+            repr_ = f"{repr_} {self._name=}"
+
+        return f"{repr_} {self._steps=}>"
```

Comment:
!notimportant - I found this a bit difficult to parse, could it be simpler? e.g.
```suggestion
        name_str = f"{self._name=} " if self._name is not None else "" 
        return f"<{type(self).__name__} {name_str}{self._steps=}>"
```

Reply:
I think I marginally prefer my approach here - fixed preamble, optional bit in the middle, fixed postamble.